pages/dashboard/github-integration.md

```diff
@@ -0,0 +1,153 @@
+# Integração com o GitHub
+
+O [Cypress Dashboard](https://on.cypress.io/dashboard) consegue integrar seus testes Cypress
+com seu Workflow do GitHub através de [verificações de status](#verificacoes-de-status)
+e comentários de [pull requests](#comentarios-de-pull-requests). 
+
+Um projeto inicialmente precisa estar [configurado para gravação](https://docs.cypress.io/guides/dashboard/projects)
+para que o Cypress Dashboard possa utilizar a integração com o GitHub.
+
+![pull-request-cypress-integration-comments-github-checks](https://docs.cypress.io/_nuxt/img/pull-request-cypress-integration-comments-github-checks.8fd68f7.jpg)
+
+>A plataforma local GitHub Enterprise atualmente não é suportada.
+
+>A Integração com o GitHub depende que o seu ambiente de integração contínua forneceça dados SHA de commit
+(típicamente via variável de ambiente). Isso não é um problema para a maioria dos usuários,
+mas se você estiver com problemas de integração com o GitHub com sua configuração de integração contínua,
+por favor, certifique-se que a informação git está sendo enviada corretamente
+seguindo [essas diretrizes](https://docs.cypress.io/guides/continuous-integration/introduction#Git-information).
```

Comment:
Precisa adicionar linha TODO acima do texto para atualizar o link quando a página for traduzida.